src/index.js

```diff
@@ -54,18 +55,27 @@ class DropzoneController extends Controller {
     );
   }
 
+  fileAdded(file) {
+    this.dispatchEvent(this.fileAddedEvent, { detail: { file: file }});
+
+    const controller = new DirectUploadController(this, file);
+    this.queue.push(controller);
+    this.runUploadQueue();
+  }
+
   handleFileAdded(file) {
     if (file.accepted) {
       if (this.fileExists(file)) {
         this.dropzone.removeFile(file);
         return;
       }
 
-      this.dispatchEvent(this.fileAddedEvent, { detail: { file: file }});
-
-      const controller = new DirectUploadController(this, file);
-      this.queue.push(controller);
-      this.runUploadQueue();
+      const isTiff = file.type === "image/tiff";
```

Comment:
does this need to be a variable or can you put this right in the conditional?

Reply:
It's a habit to extract conditionals to easier to read, but I can remove the var here

Reply:
I like that strategy when it's not immediately clear from the conditional, this one basically reads the same to me though.